app/models/fictional_caretaker.py

```diff
@@ -0,0 +1,16 @@
+from app import db
```

Comment:
Why  "fictional" caretaker? You mean your planets don't have real caretakers? 🙂 

Remember that we prefer to have a file containing a class definition to match the name of the class (allowing for differences in capitalization).